charts/pravega/templates/post-install-hooks.yaml

```diff
@@ -0,0 +1,119 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "hook.fullname" . }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded, before-hook-creation, hook-failed
+rules:
+- apiGroups:
+  - pravega.pravega.io
+  resources:
+  - "*"
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - "pods"
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - "pods/exec"
+  verbs:
+  - create
+---
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "hook.fullname" . }}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
I would suggest changing the hook type to `post-install, post-upgrade`; since what the hook does (check for ready replicas, the right versions, etc) also applies to upgrades. This way there is no need for two different hooks, the same can be used for both.
That comment applies to all the resources here

Reply:
Makes sense. Modified the hook type for all resources.